app/src/common/shared/com/igalia/wolvic/addons/adapters/AddonsListViewAdapter.kt

```diff
@@ -145,19 +145,27 @@ class AddonsManagerAdapter(
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
     internal fun bindAddon(holder: AddonViewHolder, addon: Addon) {
         val context = holder.itemView.context
-        addon.rating?.let {
+        if (addon.rating != null) {
             val userCount = context.getString(R.string.mozac_feature_addons_user_rating_count_2)
             val ratingContentDescription =
                 String.format(
                     context.getString(R.string.mozac_feature_addons_rating_content_description),
-                    it.average
+                    addon.rating!!.average
                 )
             holder.ratingView.contentDescription = ratingContentDescription
             // Android RatingBar is not very accessibility-friendly, we will use non visible TextView
             // for contentDescription for the TalkBack feature
             holder.ratingAccessibleView.text = ratingContentDescription
-            holder.ratingView.rating = it.average
-            holder.userCountView.text = String.format(userCount, getFormattedAmount(it.reviews))
+            holder.ratingView.rating = addon.rating!!.average
+            holder.userCountView.text = String.format(userCount, getFormattedAmount(addon.rating!!.reviews))
+
+            holder.ratingView.visibility = View.VISIBLE
+            holder.ratingAccessibleView.visibility = View.VISIBLE
+            holder.userCountView.visibility = View.VISIBLE
+        } else {
+            holder.ratingView.visibility = View.GONE
+            holder.ratingAccessibleView.visibility = View.GONE
+            holder.userCountView.visibility = View.GONE
```

Comment:
Hmm, why are those `visibility` updates needed?

Reply:
Similarly, we need these two commits related to visibility [1](https://github.com/Igalia/wolvic/pull/391/commits/3279ffade321cd067b8386bdf0eabbe4163f593f), [2](https://github.com/Igalia/wolvic/pull/391/commits/1be3f357d6338b2b7ed93d275cd573f9db1b5c41) to allow the AddonView to bind to the new info for the addon, otherwise hide the specific section if rating and summary info unavailable. This prevents the incorrect rating info bug.